xbmc/PlayListPlayer.cpp

```diff
@@ -514,7 +514,9 @@ void CPlayListPlayer::SetShuffle(int iPlaylist, bool bYesNo, bool bNotify /* = f
 
     if (bNotify)
     {
-      std::string shuffleStr = StringUtils::Format("%s: %s", g_localizeStrings.Get(191).c_str(), g_localizeStrings.Get(bYesNo ? 593 : 591).c_str()); // Shuffle: All/Off
+      std::string shuffleStr = StringUtils::Format(
+          "{}: {}", g_localizeStrings.Get(191).c_str(),
```

Comment:
Wow this is bad for RTL languages. Not your fault though but `{}: {}` should be a translatable string.